actions/get_dependencies/action.yaml

```diff
@@ -18,5 +18,6 @@ runs:
     - name: Get dependencies
       env:
           CACHE_LIBINT: ${{steps.cache-prepare.outputs.cache-libint}}
+          CACHE_LIBFORT: ${{steps.cache-prepare.outputs.cache-libfort}}
```

Comment:
Presumably the list of dependencies getting cached is going to grow rapidly. Can we come up with a cleaner solution? Maybe pass a list of dependencies?